Update patch level to 4.19.227

rushikeshjadhav opened the pull request:
- Fixes issue https://github.com/xcp-ng/xcp/issues/522
    - Cumulative update till 4.19.227
    - Disabled Citrix patches that were taken from upstream
    - Disabled GFS2 patches as its not support by the distro

stormi commented on SPECS/kernel-alt.spec:
```diff
@@ -59,29 +60,34 @@ Source2: SOURCES/kernel/macros.kernel
+#Patch18: 0001-gfs2-Fix-marking-bitmaps-non-full.patch
+#Patch19: 0001-gfs2-Remove-unnecessary-gfs2_rlist_alloc-parameter.patch
+#Patch20: 0001-gfs2-Pass-resource-group-to-rgblk_free.patch
+##
```
Remove this empty line?

rushikeshjadhav replied:
Done

stormi commented on SPECS/kernel-alt.spec:
```diff
@@ -915,6 +1043,12 @@ fi
+- Fixes issue #522
+- Cumulative update till 4.19.227
+- Disabled Citrix patches that were taken from upstream
+- Disabled GFS2 patches as its not support by the distro
```
Shouldn't gvt patches be mentioned too?

rushikeshjadhav replied:
Done

rushikeshjadhav committed:
Update patch level to 4.19.227
- Fixes issue https://github.com/xcp-ng/xcp/issues/522
- Cumulative update till 4.19.227
- Disabled Citrix patches that were taken from upstream
- Disabled GFS2 and gvt patches as its not support by the distro

Signed-off-by: Rushikesh Jadhav <rushikesh7@gmail.com>
